Update simple.md (#27804)
dotnet/AspNetCore.Docs: aspnetcore/security/authorization/simple.md

This would allow only authenticated users to the `AccountController`, except for the `Login` action, which is accessible by everyone, regardless of their authenticated or unauthenticated / anonymous status.

> [!WARNING]
> `[AllowAnonymous]` bypasses authorization statements. If you combine `[AllowAnonymous]` and an `[Authorize]` attribute, the `[Authorize]` attributes are ignored. For example if you apply `[AllowAnonymous]` at the controller level:

> > * Any `[Authorize]` attributes on the same controller or action methods on the controller, the authorization requirements specified by `[Authorize]` ware ignored.

Reviewer: Any authorization requirements from `[Authorize]` attributes on the same controller or action methods on the controller are ignored.

> > * Any `[Authorize]` attributes on the same controller or action methods on the controller, the authorization requirements specified by `[Authorize]` ware ignored.

Author: ```suggestion
> * Any authorization requirements from `[Authorize]` attributes on the same controller or action methods on the controller are ignored.
```

> > * Any related authentication is still be done.

Author: ```suggestion
> * Authentication middleware is not short-circuited but doesn't need to succeed.
```

> > * Any related authentication is still be done.

Reviewer: Would change to: Authentication middleware is not short-circuited but doesn't need to succeed.
